### PR TITLE
Remove redundant build instructions

### LINKED ISSUE #14380: Redundant and incomplete build instructions in grpc/src/cpp/README.md
Please answer these questions before submitting your issue. 
 
### Should this be an issue in the gRPC issue tracker?
Yes 
### What version of gRPC and what language are you using?
1.9.x 
 
### What operating system (Linux, Windows, …) and version?
N/A 
 
### What runtime / compiler are you using (e.g. python version or version of gcc)
 N/A
 
### What did you do?
Build gRPC C++ on Linux
 
### What did you expect to see?
A single source of build instruction 
 
### What did you see instead?
Multiple and inconsistent build instructions:
grpc/src/cpp/README.md
grpc/INSTALL.md
grpc/examples/cpp/README.md

### PR DESCRIPTION
**Pull request by srini100**
Fixes #14380

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -70,6 +70,15 @@ automatically try and compile the `protoc` in third_party if you cloned the
...
 installed.
 
+If it hasn't been installed, you can run the following commands to install it.
+
+```sh
+$ cd grpc/third_party/protobuf
+$ sudo make install   # 'make' should have been run by core grpc
+```
+
+Alternatively, you can download `protoc` binaries from
```
Comment (mehrdada) [inline INSTALL.md]: I suggest leaving this out. Version conflict between installed protoc and protobuf submodule has been a big source of confusion and unnecessary issues get filed.

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

File: INSTALL.md (hunk 1 above)
Comment (srini100) [reply INSTALL.md]: Should I remove both "make install" as well as download options?

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: INSTALL.md (hunk 1 above)
Comment (mehrdada) [reply INSTALL.md]: make install will install the protoc that we want (the issue is the potential version mismatch between third_party/protobuf submodule and protoc downloaded by the user). Please leave it as is. Just the download options may cause confusion.

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Pull request merged**